monitor/getShortEventStats.js

```diff
@@ -24,9 +29,19 @@ async function main(bridgeMode) {
       }
     }
   } else {
+    const unclaimedStats = {}
+    if (MONITOR_HOME_TO_FOREIGN_ALLOWANCE_LIST || MONITOR_HOME_TO_FOREIGN_BLOCK_LIST) {
+      const unclaimedPool = homeToForeignRequests
+        .filter(eventWithoutReference(homeToForeignConfirmations))
+        .filter(unclaimedHomeToForeignRequests())
+
+      unclaimedStats.unclaimed = unclaimedPool.length
```

Comment:
consider to decrease `depositsDiff` by `unclaimedStats.unclaimed` as so we can see in `depositsDiff` only the number of relay operations which the oracles must handle but did not so.